client/component/comment.js

```diff
@@ -386,7 +386,7 @@ export function ActionReport({commentKey}) {
     return <SubtleButton icon={Flag} onPress={onReport}/>
 }
 
-export function Composer({about=null, commentKey, goBackAfterPost=false, topLevel=false}) {
+export function Composer({about=null, commentKey, goBackAfterPost=false, topLevel=false, preselectedPerspectiveText=null}) {
```

Comment:
preselectedPerspective is specific to the RepresentingPerspectives feature, which this component shouldn't have to know about.

Instead I think we should just have it pass down whatever params had been passed to the composer screen.

Reply:
done